forwards_paper.tex

```diff
@@ -287,19 +304,6 @@ \subsection*{Simulation benchmarks}
 from Figure~\ref{fig:runtimes_selection} for simulations that ran to completion using both methods.}
 \end{figure}
 
-The peak RAM use in gigabytes (GB) is shown for the simulations with selection in Figure~\ref{sfig:ramsel} and for the
```

Comment:
I guess I think that we should answer the question "how much more RAM does the method use?" in the main text.  The paper is not too long (currently 16 pages incl figures).  If the point is not a terribly important one, we can make it short and put the figure in the suppmat?